gtimelogrc.example

```diff
@@ -1,7 +1,6 @@
 [gtimelog_exporter]
 jira_url = https://jira.mydomain.com/
-tempo_api = rest/tempo-timesheets/3/
-tempo_user = my-user
+tempo_url = https://api.tempo.io/core/3
```

Comment:
Use v4 of the API

https://apidocs.tempo.io/

The URLs are: 


* https://api.eu.tempo.io/4 : Use this for clients based in Europe.
* https://api.us.tempo.io/4 : Use this for clients based in the Americas.
* https://api.tempo.io/4 : Use this if your clients are based in other locations, if you are unsure where your client is installed, or if you prefer a universal path.

so the generic address will work but I would suggest using the EU endpoint
```suggestion
tempo_url = https://api.tempo.io/4/
```

Reply:
Since there is a global endpoint, I'll tend to use it over location based endpoints. I'm thinking about people going abroad for work, or simply working abroad (e.g. Akim, I don't know if he uses this script)